web/js/containers/sidebar/layer.js

```diff
@@ -81,36 +82,34 @@ class Layer extends React.Component {
     }
   }
 
+  formatDisabledTitleDate = (date, period) => {
+    var dateString;
+
+    switch (period) {
```

Comment:
This is more of a personal style preference but, a `switch` statement is maybe more verbose than necessary for a conditional with only two possible outcomes.  An if statement might be simpler:

```
if (period === 'subdaily') {
  return moment(util.parseDate(date)).format('DD MMMM YYYY HH:mm') + 'Z';
}
return moment(util.parseDate(date)).format('DD MMMM YYYY');
```

Reply:
I agree with this. I think what I had in mind using the `switch` was maybe there were additional types of formats for the time we would want to associate for various types of periods. But if it's just these two cases, then and if will definitely suffice. Thanks for reviewing - really appreciate great feedback!